modules/planning/tasks/deciders/st_bounds_decider/st_bounds_decider.cc

```diff
@@ -105,18 +109,22 @@ void STBoundsDecider::InitSTBoundsDecider(
                           frame.PlanningStartPoint().v());
 }
 
-Status STBoundsDecider::GenerateRegularSTBound(STBound* const st_bound) {
+Status STBoundsDecider::GenerateRegularSTBound(
+    STBound* const st_bound, STBound* const vt_bound) {
   // Initialize st-boundary.
   for (double curr_t = 0.0; curr_t <= st_bounds_config_.total_time();
        curr_t += kSTBoundsDeciderResolution) {
     st_bound->emplace_back(curr_t, std::numeric_limits<double>::lowest(),
                            std::numeric_limits<double>::max());
+    vt_bound->emplace_back(curr_t, std::numeric_limits<double>::lowest(),
+                           std::numeric_limits<double>::max());
   }
 
   // Sweep-line to get detailed ST-boundary.
   for (size_t i = 0; i < st_bound->size(); ++i) {
-    double t, s_lower, s_upper;
+    double t, s_lower, s_upper, v_lower, v_upper;
```

Comment:
what is the meanings of v_lower and v_upper? the speed limit of the adc?  a bit confusing here